.travis.yml

```diff
@@ -91,3 +91,6 @@ jobs:
         - cp -r packages/patternfly-4/react-docs/public/assets .public/assets
         - cp -r .out .public/patternfly-3
         - yarn run surge --project .public --domain patternfly-react.surge.sh
+
+after_deploy:
+  - .circleci/trigger_browser_tests.sh
```

Comment:
This needs to be in the deploy step. Do:
 ```
    - stage: Deploy
      name: NPM and Github Release
      install: skip
      script:
        - .circleci/release.sh
        - .circleci/trigger_browser_tests.sh
```
and be sure to `chmod +x .circleci/trigger_browser_tests.sh`.

Reply:
@redallen but it really needs to be performed after deploy. Does it make sense to create additional stage?

Reply:
Only if `.circleci/release.sh` returns 0 will our package be released to NPM for you to use, and your script will run. The `after_deploy` stage would also fire up another executor that would run for 1m 30s when we can just use that one after it releases.

Reply:
@redallen I'm mostly interested in updating this page http://patternfly-react.surge.sh/patternfly-4/. Does `.circleci/release.sh` updates it as well?

Reply:
In that case, put it after `- yarn run surge --project .public --domain patternfly-react.surge.sh`.

Reply:
@redallen thank you very much for the help